### PR TITLE
Feat: Compiles Stencil

### PR DESCRIPTION
**Pull request by JuliaPoo**
Please review the changes in particular all the commits labelled HotFix and HotFixXtreme.

This branch now compiles the stencils, but doesn't use them yet.

If this branch is to be merged, the README should be updated to require llvm to be installed to build.

Also it might be worth to look at the insane amount of warnings compilation generates

### CODE REVIEW

**Comment by JuliaPoo** [comment]
What, Github pings from commit messages??? Their UI should be more explicit with that kinda thing ugh.
Hiii Kelsey Judson what's up

**Comment by JuliaPoo** [comment]
Ah the build isnt passing. Looks like a bootstrap python version thing. But im gonna go to sleep gn

**Comment by Fidget-Spinner** [comment]
Just an FYI, I am able to build Windows release x86-64 on my computer (without JIT). Not sure what's up with boostrap python. In the first place, we shouldn't even be compiling anything at the moment right? Since we left out all the tracing stuff?

**Comment by JuliaPoo** [comment]
Ok so the reason is cuz jit.yml appears to be running the test suite i think and we cannot run the test suite bcuz we dont support a bunch of features. I'll probably just remove jit.yml entirely we can eventually write out own jit tests for pylbbv specifically.

Apart from that we should be compiling the stencils, which does build in the original workflow, and when you build normally locally.

**Comment by Fidget-Spinner** [comment]
Lets leave jit.yml in but edit it to run our tier 2 tests instead.

**Comment by Fidget-Spinner** [comment]
REMINDER TO SELF: WE NEED TO ATTRIBUTE BRANDT AS CO-AUTHOR WHEN I MERGE THIS PR.

**Review by Fidget-Spinner — APPROVED** [review]
TLDR: the changes means we lose the ability for tracing (which is used by debuggers). That's not a concern because they weren't planned to be supported anyways.
